src/legacy/ui/ui_settings/routes/integration_tests/lib/servers.ts

```diff
@@ -65,9 +79,9 @@ export function getServices() {
   const callCluster = esServer.es.getCallCluster();
 
   const savedObjects = kbnServer.server.savedObjects;
-  const savedObjectsClient = savedObjects.getScopedSavedObjectsClient();
+  const savedObjectsClient = savedObjects.getScopedSavedObjectsClient({});
 
-  const uiSettings = kbnServer.server.uiSettingsServiceFactory({
+  const uiSettings = (kbnServer.server as any).uiSettingsServiceFactory({
```

Comment:
optional: you can use Hapi's Server type if you add the `uiSettingsServiceFactory` to this extension of the type: https://github.com/joshdover/kibana/blob/cb9adf744460bebd573bc9ae241c4b19b4529e12/src/legacy/server/kbn_server.d.ts#L62

Reply:
Initially, I skipped because it doesn't give us a lot of value. Right now it will look like 
```ts
uiSettingsServiceFactory: (options: any) => any;
```
but ok, let's add